pkg/extension/wasmimage/wasmimage.go

```diff
@@ -85,3 +88,30 @@ func isWasmBinary(path string) bool {
 	}
 	return bytes.Equal(buffer, magic)
 }
+
+func registryHosts(insecure, plainHTTP bool) docker.RegistryHosts {
+	client := http.DefaultClient
+
+	if insecure {
+		client.Transport = &http.Transport{
+			TLSClientConfig: &tls.Config{
+				// this option is only enabled when the user specify the insecure flag.
+				InsecureSkipVerify: true, // nolint:gosec
+			},
+		}
+	}
+
+	return func(host string) ([]docker.RegistryHost, error) {
+		config := docker.RegistryHost{
+			Client:       client,
+			Host:         host,
+			Scheme:       "https",
+			Path:         "/v2",
```

Comment:
what's this `/v2` path..?

Reply:
good question.. I'll put a comment https://docs.docker.com/registry/spec/api/#overview